service/src/test/java/uk/gov/hmcts/reform/fpl/service/OrganisationServiceTest.java

```diff
@@ -80,21 +83,28 @@ void shouldReturnUsersFromOrganisationIfExistsInRefData() {
         when(organisationApi.findUsersByOrganisation(AUTH_TOKEN_ID, SERVICE_AUTH_TOKEN_ID, Status.ACTIVE, false))
             .thenReturn(usersInAnOrganisation);
 
-        Set<String> userIds = organisationService
-            .findUserIdsInSameOrganisation("AN");
+        Set<String> userIds = organisationService.findUserIdsInSameOrganisation("AN");
+
+        assertThat(userIds).containsExactlyInAnyOrder("40", "41");
+    }
+
+    @Test
+    void shouldNotCheckLocalMappingWhenOrganisationExistsInRefData() {
+        OrganisationUsers usersInAnOrganisation = prepareUsersForAnOrganisation();
+        when(organisationApi.findUsersByOrganisation(AUTH_TOKEN_ID, SERVICE_AUTH_TOKEN_ID, Status.ACTIVE, false))
+            .thenReturn(usersInAnOrganisation);
+
+        organisationService.findUserIdsInSameOrganisation("AN");
 
-        assertThat(userIds)
-            .containsExactlyInAnyOrder("40", "41");
+        verify(lookupSpy, never()).getUserIds(any());
     }
 
     @Test
     void shouldReturnEmptyListWhenTheLAIsNotKnownAndTheApiReturnsNotFound() {
         when(organisationApi.findUsersByOrganisation(any(), any(), any(), any()))
-            .thenThrow(new FeignException.NotFound("No organisation", REQUEST, new byte[]{}));
+            .thenThrow(new FeignException.NotFound("No organisation", REQUEST, new byte[] {}));
```

Comment:
i think it will be Forbidden. PRD wont return notfound as if organisation exists and user belongs to it there must be at lease one user, the one who executed endpoint